### ISSUE TITLE
About the results in Table 2

### ISSUE THREAD

**Issue by abcdvzz** [UCLA-VAST/GNN-DSE#2]
Hi, I am just wondering how to get the results in your paper. Since there are multiple steps, I am a little confused about how to get the result in Table 2.

I train the model with the default setting and inference with the 'test_model_state_dict.pth'. I thought I would get a similar result with the M7 model (default setting). However, I didn't find any numbers in Table 2 similar.

Could you please tell me the correct steps? In addition, will the regression and classification task get the same result in the metrics from 'Latency' to 'BRAM'? Does the 'All' in Table 2 represents the result of RMSE in the regression task? Does the 'perf' in the log file mean the latency?

**Comment by atefehsz** [comment]
The current database included here has more than 2x kernels and more than 3x points compared to when the paper was written. As the training/validation/test set will be different, it's expected to see different accuracy results than in the paper. The current database will actually give you better HLS results in the end.

The final test is more reliable if you use the `val_model_state_dict.pth` for inference. The val model hasn't seen the test set yet so it makes a more realistic case for checking the performance on the test set. The test model is the model that would have the highest accuracy on the test as it has seen the test set.

The classification model outputs probabilities of the design belonging to two classes (valid and invalid). However, the regression model predicts latency (perf) and resource utilization. 'All' is the RMSE of all the objectives in the regression task.
